.sync.yml

```diff
@@ -3,51 +3,27 @@
   required:
```

Comment:
you should run a pdksync as well, to make sure that the next pdksync will not destroy any changes you have put in